diff --git a/src/Makefile.am b/src/Makefile.am
index 1fd79288..fbc55204 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -56,7 +56,7 @@ libngraph_la_SOURCES = axis.c axis.h nconfig.c nconfig.h gra.c gra.h	\
 	common.h nhash.h nhash.c strconv.c strconv.h oio.c gettext.h	\
 	ngraph_api.c oparameter.c oparameter.h
 
-libngraph_la_CFLAGS = $(GTK_CFLAGS) $(GSL_CFLAGS) $(libngraph_cflags) -I$(top_builddir) -DCCNAME='"'$(CC)'"' $(GTKSOURCEVIEW_CFLAGS)
+libngraph_la_CFLAGS = $(GTK_CFLAGS) $(GSL_CFLAGS) $(libngraph_cflags) -I$(top_builddir) -DCCNAME="$(CC)" $(GTKSOURCEVIEW_CFLAGS)
 libngraph_la_LIBADD = gtk/libngraphui.la math/libmath.la
 libngraph_la_LDFLAGS = $(libngraph_ldflags)
 if OS_WIN32
diff --git a/src/common.h b/src/common.h
index 969f2048..02531537 100644
--- a/src/common.h
+++ b/src/common.h
@@ -51,7 +51,9 @@
 #define __VERSION__ ""
 #endif
 
-#define COMPILER_NAME (CCNAME " " __VERSION__)
+#define STRING(str) #str
+#define MACRO2STRING(str) STRING(str)
+#define COMPILER_NAME (MACRO2STRING(CCNAME) " " __VERSION__)
 
 #define MARK_TYPE_NUM 90
 
